i18n/zh/docusaurus-plugin-content-docs/current/08-plugins/vault.md

```diff
@@ -0,0 +1,147 @@
+    # Helm chart information
+    chart:
+      # name of the chart
+      chart_name: hashicorp/vault
+      # release name of the chart
+      release_name: vault
+      # k8s namespace where Vault will be installed
+      namespace: hashicorp
+      # whether to wait for the release to be deployed or not
+      wait: true
+      # the time to wait for any individual Kubernetes operation (like Jobs for hooks). This defaults to 5m0s
+      timeout: 5m
+      values_yaml: |
+        global:
+          enabled: true
+        server:
+          affinity: ""
+          ha:
+            enabled: true
+            replicas: 3
+            raft:
+              enabled: true
+              setNodeId: true
+          namespaceSelector:
+            matchLabels:
+              injection: enabled
+```
+
+Currently, except for `values_yaml`, all the parameters in the example above are mandatory.
+
+After vault pods deployed  by the plugin vault, you can follow the instructions below to initialize the vault:
+Before you follow the instructions bellow, you must install `jq` tool: a command tool to analyze json
+
+If you work on MacOS:
+```
+brew install jq
+```
+
+If you work on linux OS like CentOS:
+```
+sudo yum install jq
+```
+
+If you work on Linux OS like Ubuntu:
+```
+sudo apt-get install jq
+```
+
+
+in the  command below, the variable $NAMESPACE you should replace with "hashicorp-vault" if you not modify the namespace variable.
+Otherwise, use the namespace name you replaced
+
+1.
+```
+   # Initialize vault-0 with one key share and one key threshold.
+   kubectl exec vault-0 -n $NAMESPACE -- vault operator init -key-shares=1 -key-threshold=1 -format=json > cluster-keys.json
+```
+2 .
+  ```
+   # Display the unseal key found in cluster-keys.json
+   cat cluster-keys.json | jq -r ".unseal_keys_b64[]"
+   ```
+3.
+```
+   # Create a variable named VAULT_UNSEAL_KEY to capture the Vault unseal key.
+   VAULT_UNSEAL_KEY=$(cat cluster-keys.json | jq -r ".unseal_keys_b64[]")
+```
+4.
+```
+   # Unseal vault-0 running on the vault-0 pod.
+   kubectl exec vault-0  -n $NAMESPACE -- vault operator unseal $VAULT_UNSEAL_KEY
+```
+you will see the above command's output like this, make sure the item "Initialized" value is "true" and the item "Sealed" value is "false"
+```shell
+Key                     Value
+---                     -----
+Seal Type               shamir
+Initialized             true
+Sealed                  false
+Total Shares            1
+Threshold               1
+Version                 1.9.2
+Storage Type            raft
+Cluster Name            vault-cluster-14052440
+Cluster ID              7630cd33-2ee1-39c1-db3f-e48a6d79970a
+HA Enabled              true
+HA Cluster              https://vault-0.vault-internal:8201
+HA Mode                 active
+Active Since            2022-04-23T16:45:47.6060163Z
+Raft Committed Index    30
+Raft Applied Index      30
+```
+
+5. Initialize vault-1 and vault-2 like vault-1
```

Comment:
Initialize `vault-1` and `vault-2` like `vault-0`? Also, are we sure that we need to do three times initialization?

Reply:
> Initialize `vault-1` and `vault-2` like `vault-0`? Also, are we sure that we need to do three times initialization?

Done. Replace the last "vault-1" with "vault-0"